this is awesome, @toolness! thanks for setting this up -- LGTM! 

one small thing -- I've been using flake8 recently in python projects to keep things tidy and i really like it. how do you feel about adopting it as our linter? here's the output:

```
➜ flake8 
./entrypoint.py:19:1: E302 expected 2 blank lines, found 1
./entrypoint.py:26:1: E302 expected 2 blank lines, found 1
./entrypoint.py:40:13: E265 block comment should start with '# '
./manage.py:5:1: E302 expected 2 blank lines, found 1
./manage.py:39:34: E701 multiple statements on one line (colon) 
```

--- a/entrypoint.py
+++ b/entrypoint.py
@@ -16,12 +16,14 @@ import subprocess
 HOST_UID = os.stat('/projects').st_uid
 HOST_USER = os.environ.get('HOST_USER', 'projects_user')
 
+
 def does_username_exist(username):
     try:
         pwd.getpwnam(username)
         return True
     except KeyError:
         return False
+
 
 def does_uid_exist(uid):
     try:
@@ -37,7 +39,6 @@ if __name__ == '__main__':
             while does_username_exist(username):
                 username += '0'
             home_dir = '/home/%s' % username
-            #print "Creating username %s with UID %d" % (username, HOST_UID)
             subprocess.check_call([
                 'useradd',
                 '-d', home_dir,

--- a/manage.py
+++ b/manage.py
@@ -2,6 +2,7 @@
 import os
 import sys
 
+
 def setup_docker_sigterm_handler():
     '''
     'manage.py runserver' is not set up to deal with a SIGTERM signal,
@@ -36,7 +37,8 @@ def setup_docker_sigterm_handler():
     signal.signal(signal.SIGTERM, handler)
 
 if __name__ == "__main__":
-    if os.getcwd() == '/projects': setup_docker_sigterm_handler()
+    if os.getcwd() == '/projects':
+        setup_docker_sigterm_handler()
 
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "app.settings")
 

